### DISCUSSION TITLE
game name?

### DISCUSSION THREAD

**Discussion by yvxnce** [Melledy/Grasscutter#1112] [Q&A] [answered]
what is the game name?

**Comment by Arikatsu** [comment]
koikatsu

**Reply by Crazygamer212** [reply]
0_0

**Reply by Crazygamer212** [reply]
go to horny jail

**Comment by 4Benj** [comment]
Minecraft

**Reply by ErrorCodesForLzx** [reply]
6

**Comment by Colerar** [comment, marked as answer]
Actually, Grasscutter is a fork of [Bukkit](https://github.com/Bukkit/Bukkit).
So the game name is ***Minecraft***, the most popular anime game. :)

**Reply by CJYKK** [reply]
I'd like Spigot better :)

**Reply by NameLord** [reply]
Nah.. Fabigot is better fam..
Fabi - Fabric
got  - Spigot

**Comment by alt3ri** [comment]
Raid Shadow Legend

**Comment by Mi-nk** [comment]
You actually all are wrong, it is American Truck Simulator - Volvo Construction Equipment

**Comment by nikoflora** [comment]
You are all super wrong. its hit fps shooter video game valorant

**Comment by untold-titan** [comment]
I dunno bout you guys, but I'm pretty sure it's actually for Portal 2

**Comment by TukanDev** [comment]
you all wrong its roblox lmfao

**Comment by zhaodice** [comment]
lol, this discussion is really getting laughing of me

**Comment by nitrog0d** [comment]
this discussion is getting my balls fried

**Comment by ghost** [comment]
This game: [https://store.steampowered.com/app/1480560/Lawn_Mowing_Simulator/](https://store.steampowered.com/app/1480560/Lawn_Mowing_Simulator/)

**Comment by Tesutarin** [comment]
Doki Doki Literature Club!

**Comment by ICEY183** [comment]
OF COURSE The Legend of Zelda: Breath of the Wild

**Comment by crowity** [comment]
Roblox

**Comment by DirieSoftie** [comment]
Actually it's called [osu!](https://github.com/ppy/osu). [osu!](https://github.com/ppy/osu)  is a free-to-play rhythm game primarily developed, published, and created by Dean "peppy" Herbert. Inspired by iNiS' rhythm game Osu! Tatakae! Ouendan, it was written in C# on the .NET framework, and was released for Microsoft Windows on 16 September 2007. The game has throughout the years been ported to macOS, Linux, Android and iOS.

**Reply by lassedds** [reply]
You are indeed correct now i will 727 you hope you have a great day

**Reply by Wenti-D** [reply]
"Welcome to osu!" 300 100 100 50 X X "See you next time"

**Comment by ytfh44** [comment]
shh,do you want all of us to face DCMA?
(it's something next to h0nkai)

**Reply by CJYKK** [reply]
Just explain "DCMA" :)

**Comment by TotallyNotOndre** [comment]
Bugsnax 2

**Comment by Petomai** [comment]
I like bananas

**Comment by nitrog0d** [comment]
Whatever you are going through in your life, seek professional help. If you think going on an online chat room and escaping your problems is that easy well you’d be wrong. Your problems follow you and they affect the people you talk to. What do you gain on tormenting a GitHub thread? Of manipulating people and trying to make them turn? You gain absolutely nothing and your mental state worsens. You won’t get better if you don’t seek help.

**Reply by Petomai** [reply]
Very well said. You should be a book author.

**Reply by lilmayofuksu** [reply]
balls

**Comment by Thyrs1** [comment]
it seems pretty clear to me
Call of Duty® Modern Warfare® II  Warzone™ 2.0

**Comment by MLChinoo** [comment]
Conter Strike : Global Offensive

**Comment by ElaXan** [comment]
Cyberpunk2012

**Comment by zym234555** [comment]
You are all wrong, It's Assassin's Creed

**Comment by spz2020** [comment]
Pizza Tower

**Comment by GntHb** [comment]
It's Bloons TD6

**Comment by marcussacana** [comment]
https://store.steampowered.com/app/589420/

**Comment by nitrog0d** [comment]
we already had grasscutter game about a month ago
from an insider

**Comment by ghost** [comment]
是的,这个游戏叫做Minecraft 为什么呢?因为我用MC开服器可以启动了这个jar核心(滑稽)
